src/GliffCard.tsx

```diff
@@ -0,0 +1,82 @@
+import { CSSProperties } from "react";
+import { Card, IconButton, Paper, Typography } from "@mui/material";
+import SVG from "react-inlinesvg";
+import makeStyles from "@mui/styles/makeStyles";
+import { theme } from "./theme";
+import { icons } from "./icons";
+
+interface Props {
+  title: string;
+  el: JSX.Element;
+  cardStyle?: CSSProperties;
+  action?: {
+    icon?: string;
+    tooltip?: string;
+    onClick?: () => void;
+  };
+}
+
+const useStyles = makeStyles({
+  closeButton: {
+    position: "absolute",
+    top: "7px",
+    right: "5px",
+  },
+
+  closeIcon: { width: "15px" },
+
+  paperHeader: {
+    padding: "10px",
+    backgroundColor: theme.palette.primary.main,
+    position: "relative",
+  },
+  headerTypography: {
+    fontSize: "21px",
+    width: "240px",
+    whiteSpace: "nowrap",
+    overflow: "hidden",
+    textOverflow: "ellipsis",
+  },
+});
+
+export function GliffCard(props: Props): JSX.Element {
```

Comment:
Do we not just want to call this Card? And should the props not inherit the MUI Card props?

Reply:
Rather than Card. Dialog or Modal? It doesn't actually replace the material card component. it's kind of a new one?

Reply:
MUI has four popover things, right, what is it most close to. Or nothing at all? @joshuajsmith what do you think?

Reply:
In fact, this raises the question of why we don't use the material UI dialog, as they have one...https://mui.com/components/dialogs/#customization